plugins/filter/README.md

```diff
@@ -0,0 +1,40 @@
+# filter
+
+This plugin adds [filter](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) and [backdrop-filter](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter) utilities to Tailwind.
+
+## Usage
+
+```js
+theme: {
+  filter: {
+  blur: {
+    10: '10px',
+  },
+  saturate: {
+    0: 0,
+  },
+},
+variants: {
+  filter: ['responsive'], // defaults to []
+},
+plugins: [
+  require('@viget/tailwindcss-plugins/filter'),
+],
+```
+
+The above configuration would create the following css, as well as their responsive variants:
+
+```css
+.blur-10 {
+  filter: blur(10px);
```

Comment:
Thoughts on using this one instead? https://github.com/benface/tailwindcss-filters Just following a similar thread from one of Megan's plugins here: https://github.com/vigetlabs/tailwindcss-plugins/pull/4#issuecomment-614663240